pkg/config/draftconfig.go

```diff
@@ -24,6 +24,7 @@ type BuilderVar struct {
 	Description   string   `yaml:"description"`
 	VarType       string   `yaml:"type"`
 	ExampleValues []string `yaml:"exampleValues"`
+	NoPrompt      bool     `yaml:"noPrompt"`
```

Comment:
💡some recommendation when you have variable which are `bool` think of using it as a reader. might I suggest something like: `isPrompt` - but again I miss context but here is some more idea: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

just as a thought, please keep it if you feel strongly for it.

Reply:
i totally agree that the proposed convention is cleaner for reading. Unfortunately the default value for a bool is false, so we need the boolean to have a negative meaning in order to retain compatibility and have a default state in which a variable is prompted. We could do something like "hasPromptDisabled" to more closely fit the guidelines, but im not sure if that's much clearer since it is still a negative

Reply:
`isPromptDisabled`?

Reply:
thanks @Tatsinnit for the article link, the discussion in the comments was also very informative. They seem to be focused mostly on the accessor side of the variable, but our users would be almost exclusively on the mutator side as they are inputting their template in yaml format. In order to fit both criteria we could use an imperative for the yaml flag and an is/has prefix for the struct field ex:
```
isPromptDisabled      bool     `yaml:"disablePrompt"`
```

Reply:
sounds good.